pkg/rule/rule_manager.go

```diff
@@ -61,19 +70,52 @@ func (r *ManagedRules) Start(ctx context.Context) error {
 			ruleData, err := r.rulesUpdateFunc()
 			if err != nil {
 				r.logger.Errorf("error with rules update function for table/chain %v/%v: %v", r.ruleTarget.table.Name, r.ruleTarget.chain.Name, err)
+
+				err = r.metrics.Count(m.Prefix("manager_loop_update_func"), 1, []string{"manager_type:rule", "success:false"}, 1)
```

Comment:
For all the metrics please add tags for set name, table name and chain name where applicable. Maybe it makes sense to make this a function like `m.Prefix` and use it for everything in rule, sets and stuff down the road?

Reply:
I added a method to manager structs to generate default tags - it could live in the metrics package but it creates a dependency cycle and there would have to be different methods for each of the manager types anyway.